app/views/backend/fixed_assets/_form.html.haml

```diff
@@ -1,10 +1,19 @@
 .flash.information
   %p= :derogatory_depreciation_explanation.tl
+
+- if params[:mode] == 'sell'
```

Comment:
Not really fan of accessing `params` in views. Don't know if its sth that is *allowed* in rails.